noxfile.py

```diff
@@ -1746,13 +1746,7 @@ def test_pkgs_onedir(session):
         else:
             file_name = "pkgtests.txt"
 
-        requirements_file = os.path.join(
```

Comment:
We don't need pkg test requirement files anymore, so I should be able to delete the files in  `requirements/static/pkg` correct?

Reply:
Correct